package/kernel/linux/modules/netdevices.mk

```diff
@@ -694,6 +694,7 @@ define KernelPackage/dsa-rtl8366rb
   DEPENDS:=+kmod-dsa-realtek @!TARGET_x86 @!TARGET_bcm47xx @!TARGET_uml
   KCONFIG:= \
 	CONFIG_NET_DSA_REALTEK_RTL8366RB \
+	CONFIG_NET_DSA_REALTEK_RTL8366RB_LEDS=y \
```

Comment:
> kernel: netdevices: rtl8366: ensure CONFIG_NET_DSA_REALTEK_RTL8366RB

This can be done later. Please drop it so that we can focus on the x86 here.

Reply:
OK